acceptance/bundle/paths/pipeline_globs/script.cleanup

```diff
@@ -0,0 +1 @@
+rm -rf root/.databricks
```

Comment:
You can inline this in the original script, no need for a separate file.

Reply:
Fixed